src/commands/docs/spec/generateDocs.spec.ts

```diff
@@ -43,7 +44,14 @@ describe('sasjs doc', () => {
         defaultTarget: 'viya'
       })
 
-      await expect(doc(new Command(`doc`))).resolves.toEqual(0)
+      const { target } = await findTargetInConfiguration(
```

Comment:
would be nice to create a variable to avoid code duplications

Reply:
fixed.